package.json

```diff
@@ -72,8 +72,8 @@
     "grid-styled": "2.0.0-10",
     "lodash": "^4.17.4",
     "moment": "^2.20.1",
-    "paths-js": "^0.4.5",
-    "react-aria-tooltip": "^1.1.1",
+    "paths-js": "^0.4.7",
```

Comment:
Do we still use `path-js`? I removed it after refactoring `DeviceStatusGauge`

Reply:
That's on the v3 branch that still uses path-js.